src/MessagingThreadHistory/MessagingThreadHistory.tsx

```diff
@@ -20,6 +20,7 @@ export interface MessageData {
   timestamp?: Date;
   content: React.ReactNode;
   index: number;
+  readStatusText?: string;
```

Comment:
I wonder if we should still have a `showReadStatus` boolean prop as you'd configured earlier. And the `readStatusText` prop should exist in addition to that.

When someone specifies `showReadStatus` without a `readStatusText`, we'll fallback to "Read".

While I like the idea of consolidating props, I view `readStatusText` as a temporary prop, to be dropped once we've updated Dewey to translate internally.

Reply:
This might be awhile away, but I also think that having consumer passed text will help if ever in the future there is group messaging and we want teachers to see, "Seen by Kristen, James"

Reply:
Also, just to double check do you mean the isRead value (within MessageData) or showReadStatus (within ThreadHistory)?

Reply:
> but I also think that having consumer passed text will help if ever in the future there is group messaging and we want teachers to see, "Seen by Kristen, James"

Great point! Given that `readStatusText` will provide permanent value beyond just the temporary value of translatability, I'm okay with us relying on it alone, without `showReadStatus`.

While this might make it marginally harder to guarantee consistency in default string choice (e.g. launchpad passes "Seen" and FamPo passes "Read"), I don't think that's a big deal.

> Also, just to double check do you mean the isRead value (within MessageData) or showReadStatus (within ThreadHistory)?

Ah! I was conflating the two together a bit. I was really just thinking conceptually about the prop(s) required to display a read receipt.